README.md

```diff
@@ -1,5 +1,24 @@
 # Fluent Bit Docker Image
 
+This is [Sumo Logic fork](https://github.com/SumoLogic/fluent-bit-docker-image)
+of the [Fluent Bit Docker Image repository](https://github.com/fluent/fluent-bit-docker-image).
+
+To build new images for a specific version of Fluent Bit `X.Y.Z`,
+create an annotated tag on this repository like `vX.Y.Z-sumo-A`,
+where `X`, `Y`, `Z` and `A` are numbers.
+For example, to build images for Fluent Bit 1.8.12, run this:
+
+```sh
+git tag -am "v1.8.12-sumo-1" v1.8.12-sumo-1
+git push --tags
```

Comment:
Perhaps it would be better just to push the tag that we're interested about?

Reply:
You mean changing `git push --tags` into `git push origin v1.8.12-sumo-1`? Yeah that's probably better.